vzstorage-pd/class.yaml

```diff
@@ -4,4 +4,6 @@ metadata:
   name: virtuozzo-storage
 provisioner: kubernetes.io/virtuozzo-storage
 parameters:
-        volumePath: "/mnt/vstorage/kube/"
+  volumePath: "/mnt/vstorage/kube/"
+  secretNamespace: "default"
```

Comment:
Please change to 'kube-system' - it is a proper namespace for system-wide services. 'default' is accessible by any user.